test/incogniaApi.test.ts

```diff
@@ -138,6 +138,27 @@ describe('API', () => {
       expect(signup).toEqual(expectedResponse)
     })
 
+    it('registers a web signup', async () => {
+      const apiResponse = {
+        id: '5e76a7ca-577c-4f47-a752-9e1e0cee9e49',
+        risk_assessment: 'low_risk'
+      }
+
+      const expectedResponse = {
+        id: '5e76a7ca-577c-4f47-a752-9e1e0cee9e49',
+        riskAssessment: 'low_risk'
+      }
+
+      nock(BASE_ENDPOINT_URL)
+        .post(`/v2/onboarding/signups`)
```

Comment:
```suggestion
        .post(`/v2/onboarding/signups`, { session_token })
```

Wdyt?

https://github.com/nock/nock#specifying-request-body

Reply:
It would be: `{ session_token: 'session_token' }` write?

Ok for me...

Reply:
Yes!

I thought of putting `sessionToken` into a variable to not repeat at the API call, do you think that would be worth it?

```js
const sessionToken = 'session_token'

...
.post(`/v2/onboarding/signups`, { session_token: sessionToken })

...

const webSignup = await incogniaApi.registerWebSignup({ sessionToken })
...
```

The same about the `registers a web login` test.

Reply:
Testing these arguments ensures that the API call will receive them right